hoodie-client/src/main/java/com/uber/hoodie/table/HoodieTable.java

```diff
@@ -263,20 +269,135 @@ public HoodieActiveTimeline getActiveTimeline() {
    * @param stats List of HoodieWriteStats
    * @throws HoodieIOException if some paths can't be finalized on storage
    */
-  public void finalizeWrite(JavaSparkContext jsc, List<HoodieWriteStat> stats)
+  public void finalizeWrite(JavaSparkContext jsc, String instantTs, List<HoodieWriteStat> stats)
       throws HoodieIOException {
-    if (config.isConsistencyCheckEnabled()) {
-      List<String> pathsToCheck = stats.stream()
-          .map(stat -> stat.getTempPath() != null
-              ? stat.getTempPath() : stat.getPath())
-          .collect(Collectors.toList());
-
-      List<String> failingPaths = new ConsistencyCheck(config.getBasePath(), pathsToCheck, jsc,
-          config.getFinalizeWriteParallelism())
-          .check(MAX_CONSISTENCY_CHECKS, INITIAL_CONSISTENCY_CHECK_INTERVAL_MS);
-      if (failingPaths.size() > 0) {
-        throw new HoodieIOException("Could not verify consistency of paths : " + failingPaths);
+    rollbackFailedWrites(jsc, instantTs, stats, config.isConsistencyCheckEnabled());
```

Comment:
Calling rollback in finalizeWrite sounds weird, either we should rename `rollbackFailedWrites` to `cleanFailedWrites` or call `rollbackFailedWrites` in the constructor.

Reply:
Renamed to cleanFailedWrites